src/nft/components/collection/CollectionNfts.tsx

```diff
@@ -111,6 +121,19 @@ const ClearAllButton = styled.button`
   background: none;
 `
 
+const InfiniteScrollWrapper = styled.div`
+  padding: 0 16px;
+  @media screen and (min-width: ${({ theme }) => theme.breakpoint.sm}px) {
+    padding: 0 20px;
+  }
+  @media screen and (min-width: ${({ theme }) => theme.breakpoint.md}px) {
+    padding: 0 26px;
+  }
+  @media screen and (min-width: ${({ theme }) => theme.breakpoint.lg}px) {
+    padding: 0 48px;
+  }
```

Comment:
Can you move that out to a reusable `css` and share with `ActionsContainer`? Looks the same!

Reply:
done!